### ISSUE TITLE
Hidden content read by the screen reader [Social Share]

### ISSUE THREAD

**Issue by imbroisi** [johnsonandjohnson/Bodiless-JS#1255]
## Description

Hidden content read by the screen reader [Social Share].

### Steps to reproduce

Please see task [AESQ-7607](https://jira.jnj.com/browse/AESQ-7607).
### Expected result

Hidden content NOT readable by the screen reader [Social Share].

### Actual result

Hidden content IS readable by the screen reader [Social Share].

### Environment
  System:
    OS: macOS 10.15.7
    CPU: (4) x64 Intel(R) Core(TM) i5-3210M CPU @ 2.50GHz
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 16.13.0 - ~/.nvm/versions/node/v16.13.0/bin/node
    Yarn: 1.22.10 - /usr/local/bin/yarn
    npm: 8.1.0 - ~/.nvm/versions/node/v16.13.0/bin/npm
  Languages:
    Python: 3.7.4 - /usr/local/opt/python/libexec/bin/python

### Affected Version

Problem found in Penaten site.

This is the penaten_ca_sync's package.json:
```
{
  "name": "@sites/penaten-ca",
  "version": "0.3.1",
  "description": "",
  "author": "",
  "homepage": "",
  "license": "UNLICENSED",
  "private": true,
  "directories": {
    "lib": "lib",
    "test": "__tests__"
  },
  "dependencies": {
    "@babel/core": "^7.9.0",
    "@bodiless/backend": "0.3.1",
    "@bodiless/card": "0.3.1",
    "@bodiless/components": "0.3.1",
    "@bodiless/components-ui": "0.3.1",
    "@bodiless/core": "0.3.1",
    "@bodiless/core-ui": "0.3.1",
    "@bodiless/documentation": "0.3.1",
    "@bodiless/fclasses": "0.3.1",
    "@bodiless/filtering": "0.3.1",
    "@bodiless/gatsby-plugin-ssi": "0.3.1",
    "@bodiless/gatsby-theme-bodiless": "0.3.1",
    "@bodiless/layouts": "0.3.1",
    "@bodiless/layouts-ui": "0.3.1",
    "@bodiless/organisms": "0.3.1",
    "@bodiless/richtext": "0.3.1",
    "@bodiless/richtext-ui": "0.3.1",
    "@bodiless/search": "0.3.1",
    "@bodiless/ui": "0.3.1",
    "@bodiless/youtube": "0.3.1",
    "@bodiless/navigation": "0.3.1",
    "@bodiless/accordion": "0.3.1",
    "@bodiless/bv": "0.3.1",
    "@canvasx/canvasx-documentation": "^0.3.1",
    "@canvasx/editors": "^0.3.1",
    "@canvasx/gatsby-plugin-akamai-auth": "^0.3.1",
    "@canvasx/gatsby-plugin-jjauth": "^0.3.1",
    "@canvasx/gatsby-plugin-onetrust": "^0.3.1",
    "@canvasx/gatsby-plugin-canvasx-sitemap": "^0.3.1",
    "@canvasx/gatsby-plugin-canvasx-perf": "^0.2.4",
    "@canvasx/layout": "^0.3.1",
    "@canvasx/link": "^0.3.1",
    "@canvasx/onetrust": "^0.3.1",
    "@canvasx/penaten-ca": "^0.3.1",
    "@canvasx/elements": "^0.3.1",
    "@canvasx/image": "^0.3.1",
    "@canvasx/card": "^0.3.1",
    "@canvasx/navigation": "^0.3.1",
    "@canvasx/search": "^0.3.1",
    "@canvasx/i18n": "^0.3.1",
    "@canvasx/share": "^0.3.1",
    "@canvasx/templates": "^0.3.1",
    "@canvasx/accordion": "^0.3.1",
    "autoprefixer": "^9.8.6",
    "cross-env": "^5.2.0",
    "docsify-cli": "^4.3.0",
    "dotenv": "^8.2.0",
    "express": "^4.17.1",
    "gatsby": "~3.13.0",
    "gatsby-plugin-canonical-urls": "^3.6.0",
    "gatsby-plugin-csp": "^1.1.3",
    "gatsby-plugin-exclude": "^1.0.2",
    "gatsby-plugin-google-tagmanager": "^3.6.0",
    "gatsby-plugin-sri": "^1.1.0",
    "informed": "~3.13.1",
    "lodash": "^4.17.19",
    "ora": "^4.0.2",
    "postcss": "^8.2.4",
    "react": "^17.0.2",
    "react-dom": "^17.0.2",
    "react-helmet": "^5.2.0",
    "react-markdown": "^4.3.1",
    "react-mde": "^10.0.3",
    "tailwindcss": "^2.0.2",
    "tsconfig-paths-webpack-plugin": "^3.2.0"
  },
  "files": [
    "lib"
  ],
  "repository": {
    "type": "git",
    "url": "https://sourcecode.jnj.com/scm/asx-nbfp/canvasx.git"
  },
  "scripts": {
    "bs": "npm run build && npm run serve",
    "build": "npm-run-all build:env-vars build:tailwind-configs build:lib build:href build:component-map build:search",
    "build:full": "npm-run-all build:env-vars build:lib build:doc build:search",
    "build:lib": "gatsby build",
    "build:doc": "cross-env BODILESS_DOCS_API_ENABLED=1 bl-docs-build",
    "build:env-vars": "generate-env-vars",
    "build:tailwind-configs": "generate-tailwind-configs",
    "build:component-map": "node ./scripts/findComponents.js",
    "build:href": "node ./scripts/hreflang.js",
    "build:search": "npm run search-index",
    "build:watch": "run-p -r dev-frontend dev-backend",
    "clean": "rimraf ./public/* && rimraf ./.cache",
    "dev-backend": "node ./node_modules/@bodiless/backend/src/server.js",
    "dev-frontend": "gatsby develop",
    "init-psh": "bodiless-psh-init",
    "lint": "eslint --cache --ext .js,.jsx,.ts,.tsx ./src",
    "serve": "gatsby serve",
    "prestart": "run-s build:env-vars build:tailwind-configs build:doc",
    "start": "run-p -r dev-backend dev-frontend",
    "search-index": "create-search-index",
    "prepare": "node ./scripts/set-missing-symlinks.js"
  },
  "devDependencies": {
    "@babel/plugin-proposal-decorators": "^7.7.0",
    "@bodiless/psh": "0.3.1",
    "@types/node": "^16",
    "@typescript-eslint/eslint-plugin": "^4.0.0",
    "@typescript-eslint/parser": "^4.0.0",
    "eslint": "^7.0.0",
    "eslint-config-airbnb": "^18.2.1",
    "eslint-config-airbnb-typescript": "^14.0.1",
    "eslint-config-prettier": "^4.2.0",
    "eslint-plugin-import": "^2.20.2",
    "eslint-plugin-jest": "^23.8.2",
    "eslint-plugin-jsx-a11y": "^6.1.1",
    "eslint-plugin-prettier": "^3.0.1",
    "eslint-plugin-react": "^7.19.0",
    "gatsby-plugin-compile-es6-packages": "^1.1.0",
    "npm-run-all": "^4.1.5",
    "rimraf": "^2.6.3",
    "typescript": "^4.4.4"
  }
}

```
